webpack/assets/javascripts/react_app/components/common/forms/Select.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const Select = (props) => {
+  return (
+    <div className="clearfix">
+      <div className="form-group">
+        <label className="col-md-2">{props.label}</label>
+        <div className="col-md-4">
+          <select className="form-control"
+                  value={props.value}
+                  onChange={props.onChange}
+          >
+            <option value="">Please select</option>
```

Comment:
string not extract, I would set a default value as an argument vs hard coded text.